airbyte-workers/src/main/java/io/airbyte/workers/temporal/sync/SyncWorkflowImpl.java

```diff
@@ -72,6 +86,23 @@ public StandardSyncOutput run(final JobRunConfig jobRunConfig,
 
     final int version = Workflow.getVersion(VERSION_LABEL, Workflow.DEFAULT_VERSION, CURRENT_VERSION);
     final String taskQueue = Workflow.getInfo().getTaskQueue();
+
+    if (version > Workflow.DEFAULT_VERSION) {
```

Comment:
We need a dedicated version for that. Re-using the existing version won't work.

Reply:
oh I actually had this locally but didn't push up the change - doing that now